templates/receipt/templates/main.tpl

```diff
@@ -0,0 +1,33 @@
+{{define "simple-paragraph"}}
+    <paragraph>
+        <text-chunk font="times" font-size="9">{{.}}</text-chunk>
+    </paragraph>
+{{end}}
+{{define "table-cell-paragraph"}}
+    <table-cell align="left" border-style="none" border-width="0">
+        {{template "simple-paragraph" .}}
+    </table-cell>
+{{end}}
+<division margin="0 10 10 10" padding="5, 5, 5, 5">
+    <image src="path('./templates/res/unidoc-logo.png')" width="55.87" height="18" margin="0 0 0 0" align="center"></image>
+    <paragraph text-align="center" margin="10 0 0 0">
+        <text-chunk font="times" font-size="18"> {{.Title}} </text-chunk>    
+    </paragraph>
+    <line fit-mode="fill-width" position="relative" thickness= "1.2" margin="10 0 0 0"></line>
+</division>
+<division margin="0 0 10 10" padding="5, 5, 5, 5">
+    <paragraph text-align="left">
+        <text-chunk font="times" font-size="11">Membership fees are billed at the beginning of each period</text-chunk>
```

Comment:
Why did you create the first four text chunks instead of creating just one with more text in it? Their styles are identical.

Reply:
The ideas here is to make the template file more readable. If the whole text is in a single `text-chunk` every indented text introduces a space in the rendered text of the pdf file. Or the whole text should be in a single line which is not practical for a long text.

Reply:
I am not sure if there is any other way to achieve this though.